sdk/trace/trace_test.go

```diff
@@ -1977,25 +1977,29 @@ func TestEmptyRecordingSpanDroppedAttributes(t *testing.T) {
 	assert.Equal(t, 0, (&recordingSpan{}).DroppedAttributes())
 }
 
-func TestAddLinkWithInvalidSpanContext(t *testing.T) {
+func TestAddLinkToNonRecordingSpan(t *testing.T) {
```

Comment:
Add this test because branch with non-recording span was not covered, so test-coverage failed :)